### PR TITLE
Feature/css combinators challenge 1

### PR DESCRIPTION
**Pull request by PegliOne**
This one is easy I reckon.

### CODE REVIEW

File: css-combinators-challenge-1/styles.css
Comment (marcelamejiao) [inline css-combinators-challenge-1/styles.css:1]: Hey 
Pamela, did you add any margin to the sections? I can see you add a border, but I think also it has a margin?

**Review by marcelamejiao — COMMENTED** [review]
Please check the comments, thanks

File: css-combinators-challenge-1/styles.css
Comment (PegliOne) [reply css-combinators-challenge-1/styles.css:1]: You're right. I didn't include the margin on the left. I'll add to the body though.

Comment (PegliOne) [reply css-combinators-challenge-1/styles.css:1]: The margin's been added.

**Review by marcelamejiao — APPROVED** [review]
nice!

**Pull request merged**